lib/components/DateTime/DateTimeField.tsx

```diff
@@ -139,7 +139,7 @@ export class DateTimeField extends React.Component<DateTimeFieldProps, Partial<D
         const local = props.localTimezone;
         let invalid = false;
         let initialValue = null;
-        if (props.initialValue) {
+        if (props.initialValue || props.initialValue === '') {
```

Comment:
Please do a minor increment of the lib version in package.json and then add a description of the change in the CHANGELOG